src/Miki/Services/Guild/WeeklyResponse.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Miki.Services
+{
+    using System;
+    using Miki.Bot.Models;
+
+    public enum WeeklyStatus
+    {
+        Success,
+        UserInsufficientExp,
+        GuildInsufficientExp,
+        NoRival,
```

Comment:
To keep the status consistent with the others (`{Entity}{Reason}`) could you change it to `RivalNull`?